# PennSt Instance Configuration
---
# Display Parameters ###########################################################
# Maximum frequency with which messages will be sent past mailroom.cap_freq()
# Enter 0 for no frequency capping (not recommended - buffer lag errors)
freq_max: 30
#
# Color in r,g,b
color: # dark blue
    r: 0
    g: 0
    b: 127
#
# Height of the legend text above the car (stack individual sensors manually), m
legend_text_height: 1.5
#
# Whether this instance will dictate the displayed position of the vehicle
# -If set to true, there must be a file location for the vehicle mesh
# -If set to false, veh_mesh_resource must be None
dictate_pos: False
#
# Location of vehicle mesh file
veh_mesh_resource: None
#
################################################################################
#
# Data input parameters ########################################################
# g______ Sensor to be processed: as it appears in the MOOS alog
sensor_name: 'gPennSt'
#
# z______ Measurments to be processed: as they appear in the MOOS alog
desired_variables: #list
- 'zLat'
- 'zLong'
- 'zLatStdDev'
- 'zLongStdDev'
- 'zCourse'
#
################################################################################
#
# Survey Data ##################################################################
# Where the blender track mesh is zeroed
UTMdatum: # 2 layer nested dictionary
    E: 659300 
    N: 3607850
#
################################################################################